__tests__/index.test.js

```diff
@@ -248,19 +248,47 @@ describe(`PATCH /api/articles/:article_id tests`, () => {
     describe(`Error handling tests`, () => {
         test(`404 - Path not found for /api/topi`, () => {
             return request(app)
-            .get('/api/artivle/1')
+            .patch('/api/artivle/1')
             .expect(404)
             .then(({ body }) => {
                 expect(body.msg).toBe('Route not found');
             });
         })
-        test(`400 - No article with ID 666`, () => {
+        test(`404 - No article with ID 666`, () => {
             return request(app)
-            .get(`/api/articles/666`)
+            .patch(`/api/articles/666`)
             .expect(404)
             .then(({ body }) => {
                 expect(body.msg).toBe(`No articles with ID: 666`);
             })
         })
     })
+})
+
+describe(`DELETE /api/comments/:comment_id tests`, () => {
+    describe(`DELETE tests`, () => {
+        test('DELETE /api/comments/:comment_id, status:204, responds with an empty response body', () => {
+            return request(app)
+            .delete('/api/comments/2')
+            .expect(204)
```

Comment:
- Test - can check the amount of comments afterwards to check that it is one less.

Reply:
I like the idea of this but I'm unsure how to get the test to check the length of the array as the function doesn't return anything to check?